clients/java/signalr/src/test/java/com/microsoft/signalr/WebSocketTransportTest.java

```diff
@@ -19,7 +19,7 @@
     @Test
     public void WebsocketThrowsIfItCantConnect() {
         Transport transport = new WebSocketTransport(new HashMap<>(), new DefaultHttpClient());
-        RuntimeException exception = assertThrows(RuntimeException.class, () -> transport.start("http://www.example.com").blockingAwait(1, TimeUnit.SECONDS));
+        RuntimeException exception = assertThrows(RuntimeException.class, () -> transport.start("http://url.fake").blockingAwait(1, TimeUnit.SECONDS));
```

Comment:
```suggestion
        RuntimeException exception = assertThrows(RuntimeException.class, () -> transport.start("http://www.example.com").blockingAwait(1, TimeUnit.SECONDS));
```

`example.com` is specifically reserved [by RFC](https://tools.ietf.org/html/rfc2606#section-2). In theory, I could register `url.fake` (with the new gTLD registrations) and make this test break :). Is there a reason `example.com` was causing a problem?

Reply:
Yes, it makes the test flaky